res/css/views/settings/_ProfileSettings.scss

```diff
@@ -26,10 +26,6 @@ limitations under the License.
     height: 4em;
 }
 
-.mx_ProfileSettings_controls .mx_Field {
```

Comment:
Removed because it was added space on top of the spacing already in place. Did not serve a useful purpose here.